spec/models/user_spec.rb

```diff
@@ -63,6 +63,14 @@
 
   describe 'scopes' do
     context 'role scopes' do
+      roles_except_organizer = [
```

Comment:
I would suggest to couple this closer to the actual implementation. Right now, changing the `ROLES` constant in the users model would probably not even break this test - but it should 😉 

So how about
```ruby
roles-exept_organizer = User::ROLES.except('organizer)
```